org.eclipse.xtext.xtext.generator/src/org/eclipse/xtext/xtext/generator/serializer/SerializerFragment2.xtend

```diff
@@ -534,7 +534,7 @@ import org.eclipse.xtext.util.internal.Log
 		}
 		return '''
 			/**
-			 * �NodeModelUtils.getNode(rule).textWithoutComments.trim.replace('\n', '\n* ')�
+			 * �NodeModelUtils.getNode(rule).textWithoutComments.trim.replace('\n', '\n* ').replace('*/','*&#47;')�
```

Comment:
I see. Since this is a JavaDoc comment it allows embedding HTML so your XML-style character escaping is actually the way to go. My bad.